changelog/README.md

```diff
@@ -0,0 +1,81 @@
+# Multi-file running changelog
+
+To avoid merge conflicts in the CHANGELOG.md file we accepted the multi-file running changelog strategy.
+
+*This strategy works for the libnavui-androidauto project too. It works in the `libnavui-androidauto/changelog` directory*
+
+To follow this strategy you should create a `.md` file for every PR. Choose a directory:
+
+- `changelog/unreleased/features` for **Features** changes
+- `changelog/unreleased/bugfixes` for **Bug fixes and improvements** changes
+- `changelog/unreleased/issues` for **Known issues :warning:** changes
+- `changelog/unreleased/other` for other changes
+
+You can use anything that allow .md format in changelog files.
+
+If you have implemented several features or bugfixes you should describe all of them:
+
+```
+- Description of changes in md format
+- Description of changes in md format also
+```
+
+You can choose any name for your changelog files because the GitHub action will rename files in
+`changelog/unreleased/features` and `changelog/unreleased/bugfixes` directories to `${PR_NUMBER}.md` when you open a PR.
+
+Every push to the main or release branch Assemble changelog GitHub action will be executed:
```

Comment:
Will those commit trigger metrics run? For now it seems so, because each PR merge produces 2 commits in the main branch.

Reply:
[skip ci] tag allows skipping CircleCI jobs and GitHub Actions